protocol/storage/fault_slasher_test.go

```diff
@@ -143,6 +143,30 @@ func TestFaultSlasher_Slash(t *testing.T) {
 		assert.Equal(t, 2, ob.msgCount) // A new slashing message.
 	})
 
+	t.Run("does not slash own worker address", func(t *testing.T) {
+		getf := address.NewForTestGetter()
```

Comment:
Please improve this whole test file by distinguishing the miner actor from account actor (worker) addresses much more clearly. The miner actor addresses should not come from a signer, while the worker generally should. Change the `slasherPlumbing` to actually check the parameter to `MinerGetWorkerAddress`.